### DISCUSSION TITLE
Planned support for anonymous auth?

### DISCUSSION THREAD

**Discussion by programbo** [supabase/supabase#829] [Feature Requests] [answered]
Is there a plan to support a feature similar to Firebase's `signInAnonymously()`? It is particularly handy for onboarding users, allowing a degree of access and data persistence before they sign up, and then linking their sign-up credentials to their anonymous profile.

Thanks for your great work on an exciting platform!

**Comment by atnegrete** [comment, marked as answer]
See #683
